renovate.json

```diff
@@ -1,8 +1,7 @@
 {
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
     "extends": [
-      "config:base",
-      "helpers:pinGitHubActionDigests",
```

Comment:
Just curious... Will GitHub Action Digests still be pinned via the parent `github>platform-engineering-org/.github`? (Maybe I missed it, but I didn't see this helper being explicitly mentioned in https://github.com/platform-engineering-org/.github/blob/main/default.json)

Reply:
Ah, I think I found it in https://github.com/renovatebot/renovate/issues/16100. I guess it is part of the `config:best-practices` preset, right?

Reply:
Correct